media/base/audio_timestamp_helper.h

```diff
@@ -75,9 +78,11 @@ class MEDIA_EXPORT AudioTimestampHelper {
   // Number of frames accumulated by AddFrames() calls.
   int64_t frame_count_;
 
+#if defined(STARBOARD)
```

Comment:
Just want to double check the source of this PR, i.e. what changes are applied on top of upstream files?  We should also clearly state the detailed purpose of the PR in the commit message.

Reply:
This PR updates `//media` while keeping most Cobalt customizations. This line is a mistake - I added a guard for these lines thinking they were an unlabeled trunk customization, when they were really added in the base/net update. I can see if there's a way I can rewrite the commit history to have a more descriptive message.